src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,31 +9,22 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
 {
     [Verb("file-work-items", HelpText = "Send SARIF results to a work item tracking system such as GitHub or Azure DevOps")]
     public class FileWorkItemsOptions : SingleFileOptionsBase
-    {
+    { 
         [Option(
             "project-uri",
             HelpText = "The absolute URI of the project in which the work items are to be filed, for example https://dev.azure.com/{org}/{project} or https://github.com/{org}/{project}.",
             Required = true)]
         public string ProjectUriString { get; internal set; }
 
-        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        // ProjectUri is the URI representation of ProjectUriString above. We convert the command-line argument to the URI
+        // form as part of argument validation at runtime (in order to provide better error messages than those that 
+        // the CommandlineLibrary component provides). See FileWorkItemsCommand.ValidateOptions for more details.
         public Uri ProjectUri { get; internal set; }
 
-        [Option(
-            "personal-access-token",
-            HelpText = "TEMPORARY: Specifes the personal access used to access the project")]
-        public string PersonalAccessToken { get; internal set; }
-
         [Option(
             "group",
             HelpText = "Apply a grouping strategy to factor input log file to multiple bugs. Must be one of All, PerRunPerRule or PerRunPerTargetPerRule.",
             Default = GroupingStrategy.PerRun)]
```

Comment:
Not technically in this change, but the default grouping strategy is not one of the ones listed in the help text (or if it is, it's ambiguous between PerRunPerRule and PerRunPerTargerPerRule

Reply:
Good catch, fixed.

---
In reply to: [375983193](https://github.com/microsoft/sarif-sdk/pull/1744#discussion_r375983193) [](ancestors = 375983193)